### PR TITLE
Concentration < 4, not acceptable

### PR DESCRIPTION
**Pull request by mayabrandi**
Requested by @annaengstrom . solving issue #125

### CODE REVIEW

File: EPPs/MAF_calc_vol.py [hunk 1]
```diff
@@ -29,8 +29,8 @@ def apply_calculations(self):
         for art in self.out_analytes:
             sample_conc = art.udf.get('Concentration')
             qc_flag='FAILED'
-            if sample_conc is None:
-                logging.exception('Missing concentration') 
+            if sample_conc is None or sample_conc<4:
+                logging.exception('Low or Missing concentration')
```
Comment (annaengstrom) [inline EPPs/MAF_calc_vol.py]: Kan skriva 'Too low or Missing concentration

Comment (annaengstrom) [reply EPPs/MAF_calc_vol.py]: Visst kommer  `sample_conc<4` innebära att inget värde sätts samt att man får en röd ruta i placement map?

Comment (mayabrandi) [inline EPPs/MAF_calc_vol.py]:
```suggestion
                logging.exception('Too low or Missing concentration') 
```

**Comment by mayabrandi** [comment]
Yes! Så kommer det bli.

**Comment by annaengstrom** [comment]
Opening this again, just checked and for concentrations below 4 ng/ul, a value is given, but this should fail and not give a sample volume.

**Comment by annaengstrom** [comment]
Screen Shot 2020-03-11 at 10 25 07 AM

**Comment by annaengstrom** [comment]
Also, for concentrations over 1444 ng/ul, no value should be given, this should fail.

**Pull request merged**